crs_application/include/crs_application/common/config.h

```diff
@@ -74,13 +77,27 @@ struct ProcessExecutionConfig
   std::vector<double> joint_tolerance =
       std::vector<double>(6, (3.1416 / 180.0) * 2.0); /** @brief how close the robot needs to be to the last position in
                                                          radians */
+  Eigen::Vector3d position_path_tolerance = Eigen::Vector3d::Ones() * 0.01;
+  Eigen::Vector3d orientation_path_tolerance = Eigen::Vector3d::Ones() * 0.05;
+  Eigen::Vector3d position_goal_tolerance = Eigen::Vector3d::Ones() * 0.01;
+  Eigen::Vector3d orientation_goal_tolerance = Eigen::Vector3d::Ones() * 0.05;
+  double tool_speed = 0.05;
+  double target_force = 20;
+  double force_tolerance = 5;
+
+  bool force_controlled_trajectories = false;
+
+  std::string ur_tool_change_script;
+  bool execute_tool_change;
 };
 
 struct ScanAcquisitionConfig
 {
   std::vector<std::vector<double> > scan_poses;
   std::string tool_frame;
   bool skip_on_failure = false;
+
+  bool force_controlled_trajectories = false;
```

Comment:
We may not need to specify this for scan moves since we know there's no contact during scan

Reply:
The reason I specified this is because it determines whether or not there is a service call to the UR driver that switches the active controller to ensure that joint controlled motions are enabled. If there are no force controlled motions then there is never a need to switch that controller, but if there are force controlled motions then it is possible this service needs to be called. Additionally, if the system is running in simulation this service does not exist, and therefore this bool must be set to false.

Reply:
Since during scan acquisition we never execute force controlled trajectories could we always just call this service to set it in joint controlled mode?

Reply:
For handling the simulation case we could just create a minimalist python node that host the service server and doesn't do anything with the request and just return

Reply:
Okay, I can make this script to simplify things

Reply:
New node added and tested that only launches if sim is set to true